specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB - Managed Cassandra",
+    "description": "Azure Cosmos DB Database Service Managed Cassandra REST API",
+    "version": "2020-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/cassandraClusters": {
+      "get": {
```

Comment:
>       [](start = 0, length = 6)

Tracked resources must also support SystemData. This is a new enforcement, our team is yet to make changes for globalDatabaseAccount for the same. You would also have to do this in very near future(hopefully in early Co2). [https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc027-systemdata-support](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc027-systemdata-support)

Reply:
Added it to clusters and data centers.

Reply:
I think only Clusters need this, Data center is not a tracked resource, afaik.